packages/sanity/src/core/form/inputs/ReferenceInput/ReferenceField.tsx

```diff
@@ -88,7 +89,10 @@ export function ReferenceField(props: ReferenceFieldProps) {
   const hasErrors = props.validation.some((v) => v.level === 'error')
   const hasWarnings = props.validation.some((v) => v.level === 'warning')
 
-  const loadableReferenceInfo = useReferenceInfo(value?._ref, getReferenceInfo)
+  const loadableReferenceInfo: Loadable<ReferenceInfo> = useReferenceInfo(
```

Comment:
nit: do we need to assign the type here? `useReferenceInfo` has the same type defined as the return type, so this could be derived

Reply:
Good point, think you're right. Doesn't feel important enough to fix before merge, though :)